multiphysics/unsteady_cht/flow_cylinder.cfg

```diff
@@ -0,0 +1,156 @@
+%
+% Initial temperature for incompressible flows that include the
+% energy equation (288.15 K by default). Value is ignored if
+% INC_ENERGY_EQUATION is false.
+INC_TEMPERATURE_INIT= 288.15
+%
+% Non-dimensionalization scheme for incompressible flows. Options are
+% INITIAL_VALUES (default), REFERENCE_VALUES, or DIMENSIONAL.
+% INC_*_REF values are ignored unless REFERENCE_VALUES is chosen.
+INC_NONDIM= DIMENSIONAL
+%
+% ---- IDEAL GAS, POLYTROPIC, VAN DER WAALS AND PENG ROBINSON CONSTANTS -------%
+%
+% Fluid model (STANDARD_AIR, IDEAL_GAS, VW_GAS, PR_GAS,
+%              CONSTANT_DENSITY, INC_IDEAL_GAS)
+FLUID_MODEL= INC_IDEAL_GAS
+%
+% Specific heat at constant pressure, Cp (1004.703 J/kg*K (air)).
+% Incompressible fluids with energy eqn. only (CONSTANT_DENSITY, INC_IDEAL_GAS).
+SPECIFIC_HEAT_CP= 1004.703
+%
+% Molecular weight for an incompressible ideal gas (28.96 g/mol (air) default)
+% Incompressible fluids with energy eqn. only (CONSTANT_DENSITY, INC_IDEAL_GAS).
+MOLECULAR_WEIGHT= 28.96
+%
+% --------------------------- VISCOSITY MODEL ---------------------------------%
+%
+% Viscosity model (SUTHERLAND, CONSTANT_VISCOSITY).
+VISCOSITY_MODEL= CONSTANT_VISCOSITY
+%
+% Molecular Viscosity that would be constant (1.716E-5 by default)
+MU_CONSTANT= 1.7893e-05
+%
+% Sutherland Viscosity Ref (1.716E-5 default value for AIR SI)
+MU_REF= 1.716E-5
+%
+% Sutherland Temperature Ref (273.15 K default value for AIR SI)
+MU_T_REF= 273.15
+%
+% Sutherland constant (110.4 default value for AIR SI)
+SUTHERLAND_CONSTANT= 110.4
+
+% --------------------------- THERMAL CONDUCTIVITY MODEL ----------------------%
+%
+% Conductivity model (CONSTANT_CONDUCTIVITY, CONSTANT_PRANDTL).
+CONDUCTIVITY_MODEL= CONSTANT_PRANDTL
+%
+% Molecular Thermal Conductivity that would be constant (0.0257 by default)
+KT_CONSTANT= 0.0257
+%
+% Laminar Prandtl number (0.72 (air), only for CONSTANT_PRANDTL)
+PRANDTL_LAM= 0.72
+%
+% Turbulent Prandtl number (0.9 (air), only for CONSTANT_PRANDTL)
+PRANDTL_TURB= 0.90
+%
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+%
+% Numerical method for spatial gradients (GREEN_GAUSS, WEIGHTED_LEAST_SQUARES)
+NUM_METHOD_GRAD= GREEN_GAUSS
+%
+% ------------------------ LINEAR SOLVER DEFINITION ---------------------------%
+%
+% Linear solver or smoother for implicit formulations (BCGSTAB, FGMRES, SMOOTHER_JACOBI,
+%                                                      SMOOTHER_ILU, SMOOTHER_LUSGS,
+%                                                      SMOOTHER_LINELET)
+LINEAR_SOLVER= FGMRES
+%
+% Preconditioner of the Krylov linear solver (ILU, LU_SGS, LINELET, JACOBI)
+LINEAR_SOLVER_PREC= ILU
+%
+% Linael solver ILU preconditioner fill-in level (0 by default)
+LINEAR_SOLVER_ILU_FILL_IN= 0
+%
+% Minimum error of the linear solver for implicit formulations
+LINEAR_SOLVER_ERROR= 1E-15
+%
+% Max number of iterations of the linear solver for the implicit formulation
+LINEAR_SOLVER_ITER= 20
+%
```

Comment:
this was 5 iterations before, with this setup you get a vortex shedding, with 5 one doesn't